src/xkit.cs

```diff
@@ -4429,11 +4449,5 @@ interface NSAttributedStringDocumentReadingOptionKey {
 		[Field ("NSSourceTextScalingDocumentOption")]
 		NSString NSSourceTextScalingDocumentOption { get; }
 
-		// This field is really inside WebKit
-		[NoWatch, NoTV]
-		[iOS (13, 0)]
-		[MacCatalyst (13, 1)]
-		[Field ("NSReadAccessURLDocumentOption", "WebKit")]
-		NSString NSReadAccessUrlDocumentOption { get; }
```

Comment:
Looks like this is still available?

https://developer.apple.com/documentation/webkit/nsreadaccessurldocumentoption?language=objc

Reply:
Yea, this is weird, seems like something people would use fairly frequently. I think it's best to leave it in for now.